### PR TITLE
general-concepts/filesystem: Correct description for /opt

### PR DESCRIPTION
**Pull request by mgorny**
Signed-off-by: Michał Górny <mgorny@gentoo.org>

### CODE REVIEW

**Comment by ulm** [comment]
Are there actual examples of open-source software that installs into /opt? Won't we consider that a QA violation?

**Comment by mgorny** [comment]
What about open-source versions of software that is otherwise proprietary? Looking at my system, OpenJDK fits that definition.

File: general-concepts/filesystem/text.xml [hunk 1]
```diff
@@ -45,8 +45,9 @@ Any binary which links against a library under <c>/usr</c> must itself go into
 </p>
 
 <p>
-The <c>/opt</c> top-level should only be used for binary-only applications.
-Binary-only applications must not be installed outside of <c>/opt</c>.
+The <c>/opt</c> top-level should only be used for applications that
+do not conform to the standard filesystem layout. This paricularly includes
```
Comment (ulm) [inline general-concepts/filesystem/text.xml:49]:
```suggestion
do not conform to the standard filesystem layout. This particularly includes
```

Comment (mgorny) [reply general-concepts/filesystem/text.xml]: Fixed, thanks.

**Comment by ulm** [comment]
Merged, thanks.

**Pull request closed without merging**